### PR TITLE
V laanjana/package creation/j boss

### PR DESCRIPTION
**Pull request by v-laanjana**
Change(s):
See guidance below
Reason for Change(s):
This PR is part of template spec solution migration for solution Illusive Attack Management System.
See guidance below
Version Updated:
2.0.0
See guidance below
Testing Completed:
The link for test results is
[Sentinel Solution Migration](https://microsoftapc.sharepoint.com/:w:/r/teams/SentinelCATTeam/_layouts/15/Doc.aspx?sourcedoc=%7BDB7F17B5-D12B-461C-AD12-03E41AB8385C%7D&file=Sentinel%20Solution%20Migration%20-TemplateSpec%20-%20jboss.docx&action=default&mobileredirect=true&share=IQG1F3_bK9EcRq0SA-QauDhcAXe_7-cHkoU6coLyVIX5N8w) JBoss

See guidance below
This is an 3P solution.

### CODE REVIEW

File: Solutions/JBoss/Data/Solution_Jboss.json [hunk 1]
```diff
@@ -0,0 +1,17 @@
+{
+  "Name": "JBoss",
+  "Author": "Microsoft",
```
Comment (v-sabiraj) [inline Solutions/JBoss/Data/Solution_Jboss.json:3]: @v-laanjana, add email of author

File: Solutions/JBoss/Data/Solution_Jboss.json [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+  "Logo": "<img src=\"https://raw.githubusercontent.com/Azure/Azure-Sentinel/master/Logos/contrastsecurity_logo.svg\"width=\"75px\"height=\"75px\">",
```
Comment (v-sabiraj) [inline Solutions/JBoss/Data/Solution_Jboss.json:4]: @v-laanjana,change the logo file. It is redirecting to other logo.

File: Solutions/JBoss/Package/createUiDefinition.json [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+      {
+        "name": "dataconnectors",
+        "label": "Data Connectors",
+        "bladeTitle": "Data Connectors",
+        "elements": [
+          {
+            "name": "dataconnectors1-text",
+            "type": "Microsoft.Common.TextBlock",
+            "options": {
+              "text": "This solution installs the data connector for ingesting JBoss events into Microsoft Sentinel. After installing the solution, configure and enable this data connector by following guidance in Manage solution view. The solution installs a parser that transforms ingested data. The transformed logs can be accessed using the JBossEvent Kusto Function alias. "
```
Comment (v-sabiraj) [inline Solutions/JBoss/Package/createUiDefinition.json:63]: Please update the correct text, thanks

File: Solutions/JBoss/Package/mainTemplate.json [hunk 4]
```diff
@@ -0,0 +1,673 @@
...
+      "name": "[concat(parameters('workspace'),'/Microsoft.SecurityInsights/',variables('_dataConnectorContentId1'))]",
+      "apiVersion": "2021-03-01-preview",
+      "type": "Microsoft.OperationalInsights/workspaces/providers/dataConnectors",
+      "location": "[parameters('workspace-location')]",
+      "kind": "GenericUI",
+      "properties": {
+        "connectorUiConfig": {
+          "title": "JBoss Enterprise Application Platform",
+          "publisher": "Red Hat",
+          "descriptionMarkdown": "The JBoss Enterprise Application Platform data connector provides the capability to ingest [JBoss](https://www.redhat.com/en/technologies/jboss-middleware/application-platform) events into Azure Sentinel. Refer to [Red Hat documentation](https://access.redhat.com/documentation/en-us/red_hat_jboss_enterprise_application_platform/7.0/html/configuration_guide/logging_with_jboss_eap) for more information.",
```
Comment (v-sabiraj) [inline Solutions/JBoss/Package/mainTemplate.json:328]: Change Azure Sentinel to Microsoft Sentinel.

File: Solutions/JBoss/Data/Solution_Jboss.json (hunk 2 above)
Comment (v-laanjana) [reply Solutions/JBoss/Data/Solution_Jboss.json]: fixed

**Pull request merged**